packages/pds/src/api/app/bsky/feed/getAuthorFeed.ts

```diff
@@ -19,6 +19,10 @@ export default function (server: Server) {
         throw new AuthRequiredError()
       }
 
+      if (!author) {
```

Comment:
since we updated `parameters` to allow `required`. shouldn't this be handled in xrpc-server instead?

Reply:
Yeah there's a bunch of checks that I added prior to reworking params that I haven't removed because I dont wanna do it.

Reply:
I'm happy to clean those up once this lands 👍

Reply:
1 sec I'll go through and remove them

Reply:
Done

Reply:
kk coolio 👍 

wanted to make sure those weren't required anymore. & yeah i think it's best to do it in this PR or shortly after otherwise we'll have that cruft hanging around for a while lol 😅